templates/pages/admin/form/form_comment.html.twig

```diff
@@ -0,0 +1,165 @@
+        data-glpi-form-editor-comment-details
+        class="card"
+    >
+        <div
+            class="card-status-start bg-primary"
+            data-glpi-form-editor-active-comment-status-indicator
+        ></div>
+        <div class="card-body">
+            <div class="d-flex">
+                <i
+                    class="glpi-form-editor-question-handle ti ti-grip-horizontal cursor-grab ms-auto me-auto mt-n3 mb-n2"
+                    data-glpi-form-editor-question-handle
+                    draggable="true"
+                ></i>
+            </div>
+            {# Display comment's title #}
+            <div class="d-flex mt-n1 mb-3 align-items-center">
+                <input
+                    class="form-control content-editable-h2 mb-0"
+                    type="text"
+                    name="name"
+                    value="{{ comment is not null ? comment.fields.name|e('html_attr') : '' }}"
+                    placeholder="{{ __("Add a title...") }}"
+                    aria-label="{{ __("Comment title") }}"
+                    data-glpi-form-editor-dynamic-input
+                    data-glpi-form-editor-on-input="compute-dynamic-input"
+                    data-glpi-form-editor-comment-details-name
+                />
+                <span data-glpi-form-editor-required-mark class="text-danger d-none">*</span>
+                {# Duplicate comment #}
+                <i
+                    class="ti ti-copy ms-auto cursor-pointer"
+                    data-bs-toggle="tooltip"
+                    data-bs-placement="top"
+                    title="{{ __("Duplicate comment") }}"
+                    data-glpi-form-editor-on-click="duplicate-comment"
+                ></i>
+            </div>
+
+            {# Display common fields #}
+            <div
+                class="content-editable-tinymce"
+                data-glpi-form-editor-comment-description
+                {# Mark as secondary if empty #}
+                {{ comment is null or comment.fields.description|length == 0 ? "data-glpi-form-editor-active-comment-extra-details" : "" }}
+            >
+                {{ fields.textareaField(
+                    'description',
+                    comment is not null ? comment.fields.description : '',
+                    __('Description'),
+                    base_field_options|merge({
+                        'placeholder': __('Add a description...'),
+                        'enable_richtext': true,
+                        'add_body_classes': ['content-editable-tinymce-editor', 'text-muted'],
+                        'editor_height': "0",
+                        'rows' : 1,
+                        'toolbar_location': 'bottom',
+                        'init': comment is not null ? true : false,
+                        'mb': 'mb-0',
+                })
+                ) }}
+            </div>
+
+            <div
+                class="d-flex flex-row-reverse align-items-center mt-2"
```

Comment:
It seems this margin add a bit too much space at the end of the item compared to questions and sections.
It look more "similar" to others blocks if I remove it.

```suggestion
                class="d-flex flex-row-reverse align-items-center"
```